src/Facility.CodeGen.CSharp/CSharpGenerator.cs

```diff
@@ -269,6 +286,7 @@ private CodeGenFile GenerateDto(ServiceDtoInfo dtoInfo, Context context)
 				{
 					"System",
 					"System.Collections.Generic",
+					"System.Text.RegularExpressions",
```

Comment:
Is it unfortunate that the new generator causes every file to be modified... I should have thought of that.

One simple fix would be to move `regexFields` up here and only add it if there are any regex patterns. `WriteUsings` sorts the namespaces.